debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java

```diff
@@ -114,7 +114,7 @@ public void canSkipMessagesWithoutDebeziumCdcEnvelopeDueToMissingSchemaName() {
 
         final SourceRecord eventRouted = router.apply(eventRecord);
 
-        assertThat(eventRouted).isNull();
+        assertThat(eventRouted).isSameAs(eventRecord);
```

Comment:
The method name here (and the two below) should be adjusted accordingly.

Reply:
@gunnarmorling Done